setup.py

```diff
@@ -136,6 +136,9 @@ def run_tests(self):
         'invenio_base.apps': [
             'invenio_pidstore = invenio_pidstore:InvenioPIDStore',
         ],
+        'invenio_admin.register': [
```

Comment:
I would like to avoid a *verb* in entry point group name, hence `invenio_admin.modelviews` is fine, however one can create admin views without a SQLAlchemy model too.